xml/Microsoft.Build.Tasks/CreateProperty.xml

```diff
@@ -161,8 +161,8 @@
                         Output TaskParameter="Value" PropertyName="MyTargetsToBuild"
                     /CreateProperty
             
-            We need to respect the semicolon that they put in the value, and need to treat
-            this exactly as if they had done:
+            We need to respect the semicolon that he put in the value, and need to treat
```

Comment:
I would revert this file.

Reply:
@ghogen It looks like two of the MSBuild changes revert a bulk pronoun change from a few days ago. Should these be updated in the MSBuild source code?

Reply:
Yes, but for now let's make the change here to pass this PR.
```suggestion
            We need to respect the semicolon that they put in the value, and need to treat
```